services/weblate/weblate-project-translated-percentage.service.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { optionalUrl } = require('../validators')
+const { colorScale } = require('../color-formatters')
+
+const schema = Joi.object({
+  translated_percent: Joi.number().required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  server: optionalUrl.required(),
+}).required()
+
+const documentation = `
+  <p>
+    This badge displays the percentage of strings translated on a project on a
+    Weblate instance.
+  </p>
```

Comment:
Same theme here as the feedback provided in the other class. It's helpful for us as maintainers during initial discussion and implementation to understand what exactly the badges represent, but I feel like we shouldn't need basic descriptions like this in the help text which is what is shown to the end users trying to create a specific instance of a badge (since those are the folks that will be users of the service/know what the badge is).

If there's information like this that you think would be helpful for observers of the Shields _source code_ then I think that'd be best served as an inline comment in the file